lib/src/main/java/com/google/android/material/composethemeadapter/TypedArrayUtils.kt

```diff
@@ -157,18 +157,18 @@ internal fun parseShapeAppearance(
         when (a.getInt(R.styleable.ComposeThemeAdapterShapeAppearance_cornerFamily, 0)) {
             0 -> {
                 RoundedCornerShape(
-                    topLeft = cornerSizeTL ?: defaultCornerSize ?: fallbackShape.topLeft,
-                    topRight = cornerSizeTR ?: defaultCornerSize ?: fallbackShape.topRight,
-                    bottomRight = cornerSizeBR ?: defaultCornerSize ?: fallbackShape.bottomRight,
-                    bottomLeft = cornerSizeBL ?: defaultCornerSize ?: fallbackShape.bottomLeft
+                    topStart = cornerSizeTS ?: defaultCornerSize ?: fallbackShape.topStart,
```

Comment:
Just checking, but this isn't what I meant. The attributes are left/right so you need to convert them to start/end based on the layout direction. Currently the values are only correct in LTR, not RTL.

Reply:
Aaah my bad. I'll open a new PR

Reply:
https://github.com/material-components/material-components-android-compose-theme-adapter/pull/44